src/SlackMessages.ts

```diff
@@ -253,3 +246,97 @@ export function rugButtonFrom(action: ButtonSpec, command: IdentifiableInstructi
     }
     return button;
 }
+
+interface StandardAttachmentDefaults {
```

Comment:
Should we move these into a separate file in order to keep slack-messages more generic?